yaksh/templates/yaksh/addteacher.html

```diff
@@ -0,0 +1,83 @@
+{% extends "manage.html" %}
+
+{% block title %} Add teacher {% endblock title %}
+{% block subtitle %} {{ course.name }} {% endblock %}
+
+{% block css %}
+<link rel="stylesheet" media="all" type="text/css" href="{{ URL_ROOT }}/static/yaksh/css/course.css" />
+{% endblock %}
+
+{% block manage %}
+<center><h3>Add Teachers for this course</h3><br></center>
+<center><h3>Search teacher with username, firstname, lastname, email</h3><br></center>
+<div align="center">
+		<form action="{{ URL_ROOT }}/exam/manage/searchteacher/{{ course.id }}/" method="post">
+		{% csrf_token %}
+		Search Teacher: <input type="text" name="uname" style="height: 25px; padding: 0px"><br><br>
+		<center><button class="btn" type="submit">Search</button>
+		<button class="btn" type="button" name="button" onClick='location.replace("{{URL_ROOT}}/exam/manage/courses");'>Cancel</button> </center></form>
+</div>
+<br><br>
+<form action="{{ URL_ROOT }}/exam/manage/addteacher/{{ course.id }}/" method="post">
+{% csrf_token %}
+{% if success == True %}
+	{% if teachers|length == 0 %}
+		<center><h3>No results found</h3></center>
+	{% else %}
+		<center><b><u>Search Results</u></b></center><br>
+		<table>
+            <th></th>
+			<th>First Name</th>
+			<th>Last Name</th>
+			<th>Email</th>
+			<th>Institute</th>
+			<th>Department</th>
+            <th>Position</th>
+			{% for teacher in teachers %}
+				<tr>
+				<td><input type="checkbox" name="check" value="{{ teacher.id }}"></td>
+				<td>{{ teacher.first_name }}</td>
```

Comment:
Can display username as well because it is search by username, firstname. Will be helpful.